--- magma-2.2.0/CMakeLists.txt	2018-03-13 15:02:36.000000000 -0400
+++ magma-2.0.2/CMakeLists.txt	2018-03-13 15:03:37.000000000 -0400
@@ -4,6 +4,7 @@
 # to disable Fortran, set this to "off"
 # see also -DADD_ below
 set( USE_FORTRAN on CACHE BOOL "Fortran is required for some tester checks, but can be disabled with reduced functionality" )
+set( NO_OPENMP on CACHE BOOL "Disable openmp" )
 
 if ( USE_FORTRAN )
 	project( MAGMA C CXX Fortran )
@@ -79,14 +80,16 @@
 
 # ----------------------------------------
 # locate OpenMP
-find_package( OpenMP )
-if ( OPENMP_FOUND )
-	message( STATUS "Found OpenMP" )
-	message( STATUS "    OpenMP_C_FLAGS   ${OpenMP_C_FLAGS}" )
-	message( STATUS "    OpenMP_CXX_FLAGS ${OpenMP_CXX_FLAGS}" )
-	set( CMAKE_C_FLAGS   "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}" )
-	set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}" )
-endif()
+if( NOT NO_OPENMP )
+  find_package( OpenMP )
+  if ( OPENMP_FOUND )
+    message( STATUS "Found OpenMP" )
+    message( STATUS "    OpenMP_C_FLAGS   ${OpenMP_C_FLAGS}" )
+    message( STATUS "    OpenMP_CXX_FLAGS ${OpenMP_CXX_FLAGS}" )
+    set( CMAKE_C_FLAGS   "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}" )
+    set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}" )
+  endif()
+endif(NO_OPENMP)
 
 
 # ----------------------------------------
